includes/class-edd-customer-query.php

```diff
@@ -349,7 +349,7 @@ protected function get_items() {
 		}
 
 		// If querying for a count only, there's nothing more to do.
-		if ( $this->query_vars['count'] ) {
+		if ( $this->query_vars['count'] && isset( $items[0] ) ) {
```

Comment:
I believe that this `isset` check should be within this if statement, not part of it.

Reason being that this if statement checks if we are getting just a count, however now it will be run if we're asking for count AND there are items.

If the count is being requested but there are no items, then we should return `0` within this if statement, or if there are items, continue returning `return intval( $items[0]->count );`

Reply:
Makes sense... done.